marc.py

```diff
@@ -539,7 +552,7 @@ def create_record(cls, work, annotator, force_create=False, integration=None):
         record = None
         existing_record = getattr(work, annotator.marc_cache_field)
         if existing_record and not force_create:
-            record = Record(data=existing_record, force_utf8=True)
+            record = Record(data=existing_record.encode('utf-8'), force_utf8=True)
```

Comment:
This is unrelated to the time stuff, but I found some encoding errors while testing.